api/globalSearch/index.spec.ts

```diff
@@ -0,0 +1,66 @@
+import * as chai from 'chai';
+import { expect } from 'chai';
+import { NextFunction } from 'express';
+import 'mocha';
+import * as sinon from 'sinon';
+import * as sinonChai from 'sinon-chai';
+import { mockReq, mockRes } from 'sinon-express-mock'
+import * as globalSearchServices from './index';
+import { http } from '../../api/lib/http';
+import { Jurisdiction } from '@hmcts/ccd-case-ui-toolkit';
+import { GlobalSearchService } from '../../api/interfaces/globalSearchService';
+
+chai.use(sinonChai);
+
+describe('Jurisdiction', () => {
+
+	let sandbox: sinon.SinonSandbox;
+	let spy: any;
+	let jurisdictionList: Jurisdiction[] = [
```

Comment:
@johnbenjamin-hmcts - sorry, should've picked this up earlier... can you change to a `const`?